### ISSUE TITLE
Use package import comment

### ISSUE THREAD

**Issue by kardianos** [natefinch/npipe#14]
If you'd like people to use the gopkg.in reference, then in one of your file add:
package npipe // import "gopkg.in/natefinch/npipe.v2"

This will signal to tools and people what reference should be used.

**Comment by natefinch** [comment]
Yeah, this was written before that feature. I should definitely add that in. Thanks for reminding me!
